<ui version="4.0" >
 <class>DetailsDialog</class>
 <widget class="QDialog" name="DetailsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="leftMargin" >
    <number>6</number>
   </property>
   <property name="rightMargin" >
    <number>6</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="6" >
    <widget class="QLineEdit" name="pathLineEdit" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" >
       <property name="sizeConstraint" >
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="CoverWidget" native="1" name="coverWidget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>110</width>
           <height>110</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Tags</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <widget class="QRadioButton" name="id3v1RadioButton" >
            <property name="text" >
             <string>ID3v1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="id3v2RadioButton" >
            <property name="text" >
             <string>ID3v2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="apeRadioButton" >
            <property name="text" >
             <string>APE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title" >
        <string>Audio Properties</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QLabel" name="propertiesLabel" >
          <property name="text" >
           <string>-</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="5" >
    <widget class="QGroupBox" name="tagGroupBox" >
     <property name="title" >
      <string>ID3v1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_47" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Title:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="titleLineEdit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_43" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Artist:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="artistLineEdit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_45" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Album:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="albumLineEdit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Composer:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="composerLineEdit" />
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_46" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Genre:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLineEdit" name="genreLineEdit" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Disc number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="discSpinBox" >
          <property name="specialValueText" >
           <string>?</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_42" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Track:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QSpinBox" name="trackSpinBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="specialValueText" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_44" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Year:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="yearSpinBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="specialValueText" >
           <string>?</string>
          </property>
          <property name="suffix" >
           <string/>
          </property>
          <property name="maximum" >
           <number>2012</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_48" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Comment:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QTextBrowser" name="commentBrowser" >
        <property name="readOnly" >
         <bool>false</bool>
        </property>
        <property name="html" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;">&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>192</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="createButton" >
     <property name="text" >
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="deleteButton" >
     <property name="text" >
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QPushButton" name="saveButton" >
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" >
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoverWidget</class>
   <extends>QWidget</extends>
   <header>coverwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>DetailsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>546</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel" >
     <x>422</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
